zds/pages/views.py

```diff
@@ -79,6 +79,11 @@ def association(request):
     return render(request, "pages/association.html")
 
 
+def accessibilite(request):
```

Comment:
Notre projet fonctionne un peu bizarrement : 
- le code doit être en anglais (y compris les commentaires) ;
- tous les éléments affichés aux utilisateurs doivent être en français (principalement dans le code HTML mais aussi les messages d'erreur dans le code Python et les chaînes de caractères dans le code JS)  ;
- les noms de commits peuvent être en français ou en anglais ;
- les discussions (tickets, PR, Discord) sont en français (sauf en cas de contributeur non francophone).

Donc pour une question de cohérence du code il faudrait faire ce changement : 
```suggestion
def accessibility(request):
```